configDiffer/cli.py

```
@@ -0,0 +1 @@
+import argparse
```

Flake8 (7.2.0)

[error] 'argparse' imported but unused

(F401)